content/en/storage-providers/advanced-configurations/logging.md

```diff
@@ -6,6 +6,8 @@ draft: false
 menu:
     storage-providers:
         parent: "storage-providers-advanced-configurations"
+aliases:
+    - /storage-providers/operate/journal/
```

Comment:
This redirect won't work as long as `content/en/storage-providers/operate/journal.md` exists. Is this line intentional?

Reply:
Never mind. Just seen that the `content/en/storage-providers/operate/journal.md` page is set to draft.